test/integration/integration_test.go

```diff
@@ -29,6 +29,7 @@ const (
 	FakeImageScripts                = "sti_test/sti-fake-scripts"
 	FakeImageScriptsNoSaveArtifacts = "sti_test/sti-fake-scripts-no-save-artifacts"
 	FakeImageNoTar                  = "sti_test/sti-fake-no-tar"
+	FakeImageOnBuild                = "sti_test/sti-fake-onbuild"
```

Comment:
I had misunderstood how the integration_test had worked, addressed this by removing the unnecessary image (the one without the ONBUILD instruction)